README.md

```diff
@@ -61,13 +61,23 @@ With Extension for Scikit-learn, you can:
 
 - **Enable CPU optimizations**
 
-    ```py
+    ```python
     import numpy as np
     from sklearnex import patch_sklearn
     patch_sklearn()
 
     from sklearn.cluster import DBSCAN
 
+    X = np.array([[1., 2.], [2., 2.], [2., 3.],
+                  [8., 7.], [8., 8.], [25., 80.]], dtype=np.float32)
+    clustering = DBSCAN(eps=3, min_samples=2).fit(X)
+    ```
+  - Or without patching:
```

Comment:
I'm ok with this format or alternatively we can show one case with patching and another without , not necessary showing both patched and not patched variants